scripts/env/check-yarn

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+/**
+ * Check that yarn is available. If not, ask the user to run `corepack enable`
+ * which will provide it.
+ *
+ * Note that any yarn@>1 will do, since it will defer to
+ * the copy in that lives in .yarn/releases (which is the version we want to
+ * use).
+ */
+
+// no external deps can be used here, because this runs before deps are installed
+const { execSync } = require('child_process');
+const { stdout } = require('process');
+const { log, warn } = require('../log');
+
+const reset = '\x1b[0m';
+const blue = '\u001b[34m';
```

Comment:
Should we add a link to where we can get these magic strings, such as the [Wikipedia page for ANSI escape codes](https://en.wikipedia.org/wiki/ANSI_escape_code#3-bit_and_4-bit)?